src/edits/double_arrow.js

```diff
@@ -54,7 +54,7 @@ class DoubleArrow extends Evented {
           const text = this.element.querySelector('text')
           if (text !== null) {
```

Comment:
@ombr j'avais pas mis ça partout mais il y a pas mal de place ou c'est comme ça.
```suggestion
          if (text) {
```
Idem pour `input`.